### PR TITLE
chore(deps): loopback version update

### LINKED ISSUE #60: Loopback version update
**Describe the bug**
- Loopback version update

### PR DESCRIPTION
**Pull request by RaghavaroraSF**
## Description

- Loopback version update

GH-60

Fixes #60

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by RaghavaroraSF** [comment]
vulnerability
This one thing needed to be addressed

File: package.json [hunk 1]
```diff
@@ -45,29 +45,29 @@
     "!*/__tests__"
   ],
   "dependencies": {
-    "@loopback/boot": "^5.0.9",
-    "@loopback/core": "^4.0.9",
-    "@loopback/rest": "^12.0.9",
+    "@loopback/boot": "^5.0.10",
+    "@loopback/core": "^4.0.10",
+    "@loopback/rest": "^12.0.10",
```
Comment (shubhamp-sf) [inline package.json]: These aren't the latest versions @RaghavaroraSF please update this PR.

**Review by shubhamp-sf — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (RaghavaroraSF) [reply package.json]: Updated

**Review by shubhamp-sf — APPROVED** [review]

**Pull request merged**